thunder.profile

```diff
@@ -5,9 +5,22 @@
  * Enables modules and site configuration for a thunder site installation.
  */
 
+use Drupal\Core\Extension\Extension;
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\block\Entity\Block;
 
+/**
+ * Implements hook_system_info_alter().
+ */
+function thunder_system_info_alter(array &$info, Extension $file, $type) {
+  // Thunder can not work properly without these modules. So they are enforced
+  // to be enabled.
+  $required_modules = ['config_selector'];
+  if ($type == 'module' && in_array($file->getName(), $required_modules)) {
+    $info['required'] = TRUE;
```

Comment:
Often implementations add an explanation here too. But the default reason actually looks good. On the extend screen it says "Required by: Thunder" which is exactly the case. We might in the future want to provide more explicit reasons but adding provision for that is premature.